data/json/recipes/tools/containers.json

```diff
@@ -785,5 +785,20 @@
     "autolearn": true,
     "qualities": [ { "id": "CUT", "level": 1 } ],
     "components": [ [ [ "cardboard", 90 ] ], [ [ "duct_tape", 100 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "journal",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_CONTAINERS",
+    "skill_used": "tailor",
+    "skills_required": [ "tailor", 3 ],
+    "difficulty": 3,
+    "time": "90 m",
+    "autolearn": true,
+    "qualities": [ { "id": "SEW", "level": 1 }, { "id": "LEATHER_AWL", "level": 1 }, { "id": "CUT", "level": 1 } ],
+    "proficiencies": [ { "proficiency": "prof_leatherworking_basic", "time_multiplier": 2 } ],
+    "components": [ [ [ "leather", 3 ], [ "tanned_hide", 1 ], [ "tanned_pelt", 1 ] ], [ [ "filament", 35, "LIST" ] ] ]
```

Comment:
can there be other ways to make a journal? I've seen journals with cardboard backs and just rings to bind them together.
Unless you can add these components, I'd suggest this to be renamed to a leatherback journal and add other types in the future

Reply:
Journal is a very generic word, and can mean a lot of different ways to keep track of notes.

Reply:
I'll rename it to leather journal, however I don't plan on adding any more content to this.